src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -199,7 +199,14 @@
 
                             Sfjs.renderAjaxRequests();
                         }
-                    }, false);
+                    };
+
+                    /* check for older browsers */
+                    if (!window.addEventListener) {
```

Comment:
you should use ``this.addEventListener`` here as this is what you use just after